crates/hyperswitch_connectors/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "hyperswitch_connectors"
+version = "0.1.0"
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+
+[features]
+frm = ["hyperswitch_domain_models/frm", "hyperswitch_interfaces/frm"]
+payouts = ["hyperswitch_domain_models/payouts", "api_models/payouts", "hyperswitch_interfaces/payouts"]
+
+[dependencies]
+async-trait = "0.1.79"
+error-stack = "0.4.1"
+serde = { version = "1.0.197", features = ["derive"] }
+serde_json = "1.0.115"
+strum = { version = "0.26", features = ["derive"] }
+
+# First party crates
+api_models = { version = "0.1.0", path = "../api_models", features = ["errors"] , default-features = false}
+cards = { version = "0.1.0", path = "../cards" }
```

Comment:
Remove this dependency if unused

Reply:
This is being used in `helcim::transformers`